src/credentials/FabricTable.cpp

```diff
@@ -19,11 +19,12 @@
  * @brief Defines a table of fabrics that have provisioned the device.
  */
 
+#include "FabricTable.h"
```

Comment:
We should use full include path instead of relative path.

Reply:
@andy31415 had previously asked to use this form of include in another PR. Happy to do whatever is the norm nowadays.

Reply:
Either should work, would not block the PR on it. Generally I am unsure if 'full path' is actually defined. Like in this case would it be 'credentials/FabricTable' or 'src/credentials/FabricTable'?

The same dir include pattern seems to be quite used:

```sh
❯ rg 'include "[^/]*"' src | cut -f1 -d: | uniq | wc -l
452
rg 'include "[^/]*"' src | egrep '.*/(.*)\.cpp:.*"\1.h"' | wc -l
```

I also cannot find the reverse of 'use full path' but maybe my grep is wrong:

```
rg 'include <' src | egrep '.*/(.*)\.cpp:.*\1"'
```

I tried to look manually and could not find any

Reply:
Based  on my basic check, I am quite convinced we should always use relative include for 'self' and never attempt for a full path. Agree full path on anything else.

Reply:
We should solve it in another place.

Reply:
Somehow my check was wrong. @kghost noticed:

```sh
❯ rg 'include <' src | egrep '.*/(.*)\.cpp:.*\1\.h' | wc -l
155
```

so to me this is about  a 50/50 split. There is no clear standard so until we have one I would not comment on it anymore - PR author decides as this seems to be a personal preference rather than style.